WordPress/Classes/WordPress.xcdatamodeld/WordPress 87.xcdatamodel/contents

```diff
@@ -572,6 +572,11 @@
         <attribute name="userID" optional="YES" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="NO" syncable="YES"/>
         <relationship name="blog" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="Blog" inverseName="connections" inverseEntity="Blog" syncable="YES"/>
     </entity>
+    <entity name="PublicizeConnectionStatsRecordValue" representedClassName=".PublicizeConnectionStatsRecordValue" parentEntity="StatsRecordValue" syncable="YES">
+        <attribute name="followersCount" attributeType="Integer 64" usesScalarValueType="YES" syncable="YES"/>
+        <attribute name="iconURLString" optional="YES" attributeType="String" syncable="YES"/>
+        <attribute name="name" attributeType="String" syncable="YES"/>
```

Comment:
In `PublicizeConnectionStatsRecordValue+CoreDataProperties.swift`, `name` is of type `String?`.
`name` is not designated optional here in the same fashion as `iconURLString`.

Do you think it would be worthwhile to reconcile that difference?

Reply:
@jklausa cleared up a similar discussion in #11010 , so this comment is now invalid.